src/stores/Frame/FrameStore.ts

```diff
@@ -604,6 +609,29 @@ export class FrameStore {
         return false;
     }
 
+    @computed get intensityConfig(): IntensityConfig {
+        let config: IntensityConfig = {nativeIntensityUnit: this.headerUnit};
+        const beams = this.beamAllChannels;
+        if (beams?.length) {
+            config["bmaj"] = beams.map(b => b?.majorAxis);
+            config["bmin"] = beams.map(b => b?.minorAxis);
+            if (this.spectralAxis?.type?.code === "FREQ") {
+                config["freqGHz"] = this.channelInfo?.values.map(x => GetFreqInGHz(this.spectralAxis.type.unit, x));
+            } else if (this.spectralAxis?.type?.code === "VRAD") {
+                config["freqGHz"] = this.channelInfo?.values.map(x => {
+                    const frequency = frequencyFromVelocity(x, this.restFreqStore?.customRestFreq?.value);
+                    return GetFreqInGHz(this.restFreqStore?.headerRestFreq?.unit, frequency);
```

Comment:
Change `headerRestFreq` to `customRestFreq` for consistency. Other than this, the PR is ready for merge. 👍
Please update the change log, thanks.